src/QAT/qat/purr/compiler/runtime.py

```diff
@@ -78,6 +79,12 @@ def _transform_results(
         Transform the raw results into the format that we've been asked to provide. Look
         at individual transformation documentation for descriptions on what they do.
         """
+
+        verification_engine = import_verification_engine()
+
+        if isinstance(self.engine, verification_engine):
+            return results
+
```

Comment:
You definitely don't want to punch a hole in the abstractions, it either passes or throws, results will just be empty.